debian/cloud-init.postinst

```diff
@@ -327,6 +327,22 @@ fix_lp1889555() {
     db_set grub-pc/install_devices_empty "false"
 }
 
+change_cloud_init_output_permissions() {
+    # As a consequence of LP: #1918303
+    local oldver="$1" last_bad_ver="21.1-17-gc6726c2b-0ubuntu1"
```

Comment:
Not sure I set this last bad version correctly. I went one back from the fix commit and determined the version from that.

Reply:
So I expect this would work, but normally I think we'd use the last version in which this was actually present in this release: `21.1-0ubuntu1` in this case. (I am also realising that cherry-picking this commit without modification won't work: I think we'll want to have the exact version for each release as `last_bad_ver`.)

Found via `rmadison cloud-init`, which produces:

```
 cloud-init | 0.6.3-0ubuntu1             | precise         | source, all
 cloud-init | 0.6.3-0ubuntu1.25          | precise-updates | source, all
 cloud-init | 0.7.5-0ubuntu1             | trusty          | source, all
 cloud-init | 0.7.5-0ubuntu1.18          | trusty-security | source, all
 cloud-init | 0.7.5-0ubuntu1.23          | trusty-updates  | source, all
 cloud-init | 0.7.7~bzr1212-0ubuntu1     | xenial          | source, all
 cloud-init | 18.2-14-g6d48d265-0ubuntu1 | bionic          | source, all
 cloud-init | 20.1-10-g71af48df-0ubuntu5 | focal           | source, all
 cloud-init | 20.3-15-g6d332e5c-0ubuntu1 | groovy          | source, all
 cloud-init | 20.4.1-0ubuntu1~16.04.1    | xenial-updates  | source, all
 cloud-init | 20.4.1-0ubuntu1~18.04.1    | bionic-updates  | source, all
 cloud-init | 20.4.1-0ubuntu1~20.04.1    | focal-updates   | source, all
 cloud-init | 20.4.1-0ubuntu1~20.10.1    | groovy-updates  | source, all
 cloud-init | 21.1-0ubuntu1              | hirsute         | source, all
```